spec/requests/notifications_controller_spec.rb

```diff
@@ -286,16 +268,6 @@ def delete_notification(resp_code, matcher)
             )
           end
 
-          it "doesn't include reviewables when navigation menu is legacy" do
-            SiteSetting.navigation_menu = "legacy"
```

Comment:
Likewise, this seems to relate to the navigation menu. Should this diff be part of this PR?